ArduCopter/Parameters.cpp

```diff
@@ -429,14 +429,12 @@ const AP_Param::Info Copter::var_info[] = {
     // @Description: Converts pilot roll and pitch into a desired rate of rotation in ACRO and SPORT mode.  Higher values mean faster rate of rotation.
     // @Range: 1 10
     // @User: Standard
```

Comment:
Shouldn't you also remove the parameter metadata comments?

Reply:
Agreed, we should remove old param metadata.

Reply:
I understand that Randy leaves them in. I will see what he says.

Reply:
Last time I checked MP beta used param descriptions from "latest" while MP stable used param descriptions from ArduPilot stable (i.e. flight code stable). I'm unsure if this has changed since but let's leave the older param descriptions in place until we know.